java/client/src/test/java/glide/api/RedisClientTest.java

```diff
@@ -1681,6 +1683,106 @@ public void lpopCount_returns_success() {
         assertEquals(value, payload);
     }
 
+    @SneakyThrows
+    @Test
+    public void lpos() {
+        // setup
+        String[] set = {"a", "b", "c", "a", "c"};
+        String element = "b";
+        String[] args = new String[] {"set", "element"};
```

Comment:
looks like set and element are not used so we can probably remove, also can we change the key name since it is a list instead of a set? Same suggestions for the other tests in this file 
```suggestion
        String[] args = new String[] {"list", "element"};
```

Reply:
i will need to keep String[] set and String element in because it's the name of the key element needed for the command to work in redis. I will change the name from String[] set to String[] list though

Reply:
You do need the `String[] args = new String[] {"list", "element"};` line, but as far as I can tell the `set` and `element` variables are assigned but never used anywhere, so they can be removed.